etna/core/blocks/contact.py

```diff
@@ -0,0 +1,24 @@
+from django.conf import settings
+
+from wagtail import blocks
+
+from .paragraph import APIRichTextBlock
+
+
+class ContactBlock(blocks.StructBlock):
+    title = blocks.CharBlock()
+    address = APIRichTextBlock(
```

Comment:
Does this need to be a RichTextBlock? Shouldn't it just be a TextField?

Reply:
Good point, I just always default to RichText 🤦‍♂️